data_override/data_override.F90

```diff
@@ -30,10 +30,13 @@ module data_override_mod
                                    get_external_field_size_classic=>get_external_field_size, &
                                    set_override_region_classic=>set_override_region, &
                                    reset_src_data_region_classic=>reset_src_data_region
-use time_interp_external2_mod, only:time_interp_external_init, time_interp_external, &
-                                   init_external_field, get_external_field_size, &
+use time_interp_external2_mod, only:time_interp_external_init_new=>time_interp_external_init, &
+                                   time_interp_external_new=>time_interp_external, &
+                                   init_external_field_new=>init_external_field, &
+                                   get_external_field_size_new=>get_external_field_size, &
+                                   set_override_region_new=>set_override_region, &
+                                   reset_src_data_region_new=>reset_src_data_region, &
```

Comment:
*new* it too relative.  If this works with fms2io, why not use that instead of "new"?

Reply:
updated in d794967226b7c7dc20e8b7ea6a8890cdd7c39478